.github/workflows/database-backup.yml

```
@@ -21,7 +21,7 @@ jobs:
       - run: exit 0
 
   backup-database:
-    if: github.event_name != 'push'
+    if: github.event_name != 'push' && secrets.DB_HOST != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)